CallTracking.Web.Test/Controllers/DashboardControllerTest.cs

```diff
@@ -8,15 +10,21 @@
 
 namespace CallTracking.Web.Test.Controllers
 {
-    public class DashboardControllerTest
+    class DashboardControllerTest
```

Comment:
It's better to be explicit about the access modifier of a method, this one should be _public_.